test/colony-task-work-rating.js

```diff
@@ -217,10 +217,9 @@ contract("Colony Task Work Rating", () => {
     it("should assign rating 5 to manager and penalise worker by 0.5, when they haven't submitted rating on time", async () => {
       const taskId = await setupAssignedTask({ colonyNetwork, colony });
       await colony.submitTaskWorkRating(taskId, WORKER_ROLE, RATING_2_SECRET, { from: EVALUATOR });
-      await forwardTime(SECONDS_PER_DAY * 5, this);
+      await forwardTime(SECONDS_PER_DAY * 6, this);
```

Comment:
I'd prefer `SECONDS_PER_DAY * 5 + 1` to `SECONDS_PER_DAY*6` to make it clearer the boundary that is being skipped over here.